libs/backend/src/cast_vote_records/build_cast_vote_record.ts

```diff
@@ -585,11 +607,6 @@ export function buildCastVoteRecord({
             electionOptionPositionMap,
           }),
         ],
-    BallotImage: imageFileUris
-      ? imageFileUris.map((imageFileUri) => ({
-          '@type': 'CVR.ImageData',
-          Location: imageFileUri,
-        }))
-      : undefined,
+    BallotImage: images ? images.map(buildCvrImageData) : undefined,
   };
```

Comment:
```suggestion
    BallotImage: images?.map(buildCvrImageData),
```

Reply:
Leaving such comments in VS Code does not always translate well. I think you can see what I meant.